### DISCUSSION TITLE
Bundle size is too large for antd. Any way to fix?

### DISCUSSION THREAD

**Discussion by salekin02** [ant-design/ant-design#50875] [Q&A]
using react with vite and antd library. after build, the bundle size is too large. mostly the icons taking huge space and the other components I did not use. about 9mb is the file size.

[visualized page](https://filetransfer.io/data-package/DdH6LEyo#link)

**Comment by mahmudulhasan-xs** [comment]
it looks like tree shaking not working

**Comment by afc163** [comment]
Could you provide a reproduce?

**Reply by salekin02** [reply]
here is visualized file by vite-visualizer:

[https://filetransfer.io/data-package/DdH6LEyo#link](https://github.com/ant-design/ant-design/discussions/visualized)

**Reply by afc163** [reply]
Could you provide a github repo to reprodcue?
